internal/infrastructure/kubernetes/proxy/resource.go

```diff
@@ -143,10 +143,16 @@ func expectedProxyContainers(infra *ir.ProxyInfra, deploymentConfig *egcfgv1a1.K
 		fmt.Sprintf("--log-level %s", logLevel),
 		"--cpuset-threads",
 	}
+
 	if componentLogLevel := componentLogLevelArgs(proxyLogging.Level); componentLogLevel != "" {
 		args = append(args, fmt.Sprintf("--component-log-level %s", componentLogLevel))
 	}
 
+	if infra.Config != nil &&
```

Comment:
can we keep this closer to L144 near `cpuset-threads` ?

Reply:
okay, done!